WTWSMS/history/characters/adnanite.txt

```diff
@@ -592,273 +493,38 @@
 163003={
 	name="Marwan"
 	dynasty = 597
-	religion="sunni"
+	religion="islamic"
 	culture="adnanite"
-	father = 163019
+	father = 36489
 	trait=sayyid
 	623.1.1={
 		birth="623.1.1"
 	}
+	632.6.8 = {
+		religion = sunni
+	}
 	685.5.7={
 		death="685.5.7"
 	}
 }
 
-163004 = {
-	name="Abd Manaf"
-	dynasty=7296
-	religion="semitic_pagan"
-	culture="adnanite"
-	father = 163005
-	444.1.1={
-		birth="444.1.1"
-	}
-	487.1.1={
-		death="487.1.1"
-	}
-}
-
-163005 = {
-	name="Qusai"
-	dynasty=7296
-	religion="semitic_pagan"
-	culture="adnanite"
-	father = 163006
-	400.1.1={
-		birth="400.1.1"
-	}
-	480.1.1={
-		death="480.1.1"
-	}
-}
-
-163006 = {
-	name="Kilab"
-	dynasty=7296
-	religion="semitic_pagan"
-	culture="adnanite"
-	father = 163007
-	373.1.1={
-		birth="373.1.1"
-	}
-	423.1.1={
-		death="423.1.1"
-	}
-}
-
-163007 = {
-	name="Murrah"
-	dynasty=7296
-	religion="semitic_pagan"
-	culture="adnanite"
-
-	333.1.1={
-		birth="333.1.1"
-	}
-	393.1.1={
-		death="393.1.1"
-	}
-}
-
-163008 = {
-	name="Taym"
-	dynasty=7296
-	religion="semitic_pagan"
-	culture="adnanite"
-	father = 163007
-	375.1.1={
-		birth="375.1.1"
-	}
-	423.1.1={
-		death="423.1.1"
-	}
-}
-
-163009 = {
-	name="Saad"
-	dynasty=7296
-	religion="semitic_pagan"
-	culture="adnanite"
-	father = 163008
-	405.1.1={
-		birth="405.1.1"
-	}
-	453.1.1={
-		death="453.1.1"
-	}
-}
-
-163010 = {
-	name="Kaab"
-	dynasty=7296
-	religion="semitic_pagan"
-	culture="adnanite"
-	father = 163009
-	435.1.1={
-		birth="435.1.1"
-	}
-	483.1.1={
-		death="483.1.1"
-	}
-}
-
-163011 = {
```

Comment:
36463?
Not sure, this one is "Umru" and 36463 is "Amr", are these names the same?

Reply:
I think this is just different spelling, their parents and children are match up.

Reply:
Yeah it's probably that, IMO we should use "Amr" name for him because it's more popular.

Reply:
> 
> 
> Yeah it's probably that, IMO we should use "Amr" name for him because it's more popular.

Ok, i will change that.

Reply:
Maybe it's already like this, unless you copied the vanilla name to Enlil version.

Reply:
Umru from vanilla has been deleted and Enlil's Amr remained, so its fine.